src/sources/logplex.rs

```diff
@@ -108,6 +121,27 @@ fn decode_message(body: Bytes, header_map: HeaderMap) -> Result<Vec<Event>, Erro
     Ok(events)
 }
 
+fn add_query_params(
+    mut events: Vec<Event>,
+    query_params_config: &[String],
+    query_params: HashMap<String, String>,
+) -> Vec<Event> {
+    for query_param_name in query_params_config {
+        let value = query_params
+            .get(query_param_name)
+            .map(String::as_bytes)
+            .unwrap_or_default();
```

Comment:
I'm seeing that this matches `add_headers`, but I'm wondering if we shouldn't map `None` to `event::Value::Null` rather than the empty string. That would help distinguish parameters that are missing from those that exist, but are empty. I think `add_headers` may have predated `event::Value::Null`; I wouldn't mind seeing that updated too.